Dockerfile

```diff
@@ -6,47 +6,58 @@ FROM ${BUILD_FROM}
 
 # Synchronize with homeassistant/core.py:async_stop
 ENV \
-    S6_SERVICES_GRACETIME=240000
+    S6_SERVICES_GRACETIME=240000 \
+    UV_EXTRA_INDEX_URL="https://wheels.home-assistant.io/musllinux-index/" \
+    UV_NO_CACHE=true \
```

Comment:
Can be removed when https://github.com/home-assistant/docker-base/pull/260 is merged